compiler/noirc_evaluator/ssa_test/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "ssa_test"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
Please look at the root Cargo.toml and the ones from other packages and do them in the same style